app/views/candidate_interface/degrees/degree/new_enic.html.erb

```diff
@@ -1,49 +1,25 @@
 
-    <p class="govuk-body">You can get a statement from <%= t('service_name.enic.full_name') %> which shows how your qualifications compare to UK qualifications. Not all providers need this.</p>
-    <%= f.govuk_radio_buttons_fieldset :have_enic_reference, legend: { text: t('application_form.degree.enic_statement.label'), size: 'm' } do %>
-      <%= f.govuk_radio_button :have_enic_reference, 'Yes', label: { text: 'Yes' }, link_errors: true do %>
-        <%= f.govuk_text_field(
-          :enic_reference,
-          label: { text: t('application_form.degree.enic_reference.label'), size: 's' },
-          hint: { text: t('application_form.degree.enic_reference.hint_text') },
-          spellcheck: false,
-        ) %>
-        <%= f.govuk_radio_buttons_fieldset(
-          :comparable_uk_degree,
-          legend: { text: t('application_form.degree.comparable_uk_degree.label'), size: 's' },
-          hint: { text: t('application_form.degree.comparable_uk_degree.hint_text') },
-        ) do %>
-          <% ApplicationQualification.comparable_uk_degrees.values.each_with_index do |value, i| %>
-            <%= f.govuk_radio_button(
-              :comparable_uk_degree,
-              value,
-              label: { text: t("application_form.degree.comparable_uk_degree.values.#{value}") },
-              link_errors: i.zero?,
-            ) %>
-          <% end %>
-        <% end %>
-      <% end %>
-      <%= f.govuk_radio_button :have_enic_reference, 'No', label: { text: 'No' } do %>
-        <p class="govuk-body">
-          You should <%= govuk_link_to('apply for a statement of comparability from UK ENIC (opens in new tab)',
-                                                            t('service_name.enic.statement_of_comparability_url'),
-                                                            target: '_blank',
-                                                            rel: 'noopener') %>.
-        </p>
+    <p class="govuk-body"> <%= t('application_form.degree.enic_statement.apply_enic') %></p>
+    <p class="govuk-body"> <%= govuk_link_to(t('application_form.degree.enic_statement.enic_link'), t('service_name.enic.statement_of_comparability_url')) %></p>
+    <p class="govuk-body"> <%= t('application_form.degree.enic_statement.percentage_text_html') %></p>
+    <p class="govuk-body"> <%= t('application_form.degree.enic_statement.enic_cost') %></p>
 
-        <p class="govuk-body">Applications with a statement are 28% more likely to be successful.</p>
-      <% end %>
+    <%= f.govuk_radio_buttons_fieldset :enic_details, legend: { text: t('application_form.degree.enic_statement.label'), size: 'm' } do %>
+      <%= f.govuk_radio_button :enic_reason, 'obtained', label: { text: t('application_form.degree.enic_statement.yes_enic') }, link_errors: true %>
+      <%= f.govuk_radio_button :enic_reason, 'waiting', label: { text: t('application_form.degree.enic_statement.waiting_for_enic') } %>
```

Comment:
Combining the value for `enic_reason` and the key in the translations would make this better too.

```ruby
<% EnicReasons.each do |enic_reason| %> # something that knows what all the valid enic_reasons are 
  <%= f.govuk_radio_button :enic_reason, enic_reason, label: { text: t("application_form.degree.enic_statement.#{enic_reason}") }, link_errors: true %>
<% end %>
```